charts/orchestrator/templates/sonataflow-service-discovery-rbac.yaml

```diff
@@ -1,6 +1,7 @@
+{{- if .Values.sonataFlowOperator.enabled }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
+kind: Role
```

Comment:
Note to myself: remove this one when we move to OSL 1.32 operator.
(I think this one will no longer be needed)